src/main/java/com/solvd/carina_testing/BrowsePage.java

```diff
@@ -0,0 +1,47 @@
+package com.solvd.carina_testing;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class BrowsePage extends AbstractPage {
+    public BrowsePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[@resource-id='com.todoist:id/content']/*/*/android.widget.Button")
+    private ExtendedWebElement settingsButton;
+
+    @FindBy(xpath = "//*[@resource-id='com.todoist:id/recycler_view']/*/android.widget.RelativeLayout")
+    private List<ExtendedWebElement> settingsList;
```

Comment:
looks like those elements are located on other page, but you can leave it like that now
and this locator could be simplified